admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/presentation/screen/deposit/mapper/countryMapper.kt

```diff
@@ -0,0 +1,23 @@
+package net.thechance.mena.admin_panel.presentation.screen.deposit.mapper
+
+import net.thechance.mena.admin_panel.domain.model.Country
+import net.thechance.mena.admin_panel.presentation.screen.deposit.DepositScreenState
+
+fun DepositScreenState.CountryUiState.toEntity(): Country {
+    return Country(
+        name=name ,
+        phoneNumberRegex=phoneNumberRegex,
+        callingCode = callingCode,
+        countryCodeName = countryCodeName,
+        flagEmoji = flagEmoji
+    )
+}
+fun Country.toUi(): DepositScreenState.CountryUiState {
```

Comment:
```suggestion
fun Country.toUiState(): DepositScreenState.CountryUiState {
```

+ this file can be extracted from the mapper login and kept inside the deposit screen package like other screens